collectors/UcsmCRCFaultCollector.py

```diff
@@ -0,0 +1,41 @@
+import urllib
+from prometheus_client.core import GaugeMetricFamily
+from BaseCollector import BaseCollector
+from ucsmsdk.ucsconstants import NamingId
+from ucsmsdk.ucsexception import UcsException
+
+
+class UcsmCRCFaultCollector(BaseCollector):
+    def __init__(self, creds, config):
+        super().__init__(creds, config)
+
+    def describe(self):
+        yield GaugeMetricFamily("ucsm_crc_error", "ucsm_crc_collector")
+
+    def collect(self):
+        print("UcsmCRCFaultCollector: Get updated handles !")
```

Comment:
can we remove this? this will end up being very spammy in the long run. if you want it for debugging, introduce a --debug or -v flag
Please do this with the other collectors too.
Please also have a look at get_handles, this looks very talkative too.

Reply:
Removed unwanted debug. Will work to add debug messages later.